Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -546,50 +546,75 @@ function ViewPanelAmenities( data:table )
 
     g_kAmenitiesIM:ResetInstances();
 
-    --Luxuries
+    -- Luxuries
     CQUI_BuildAmenityBubbleInstance("ICON_IMPROVEMENT_BEACH_RESORT", data.AmenitiesFromLuxuries, "LOC_PEDIA_RESOURCES_PAGEGROUP_LUXURY_NAME");
-    --Civics
+    
+    -- Civics
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_CIVICS") then
         CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_CONSIDER_GOVERNMENT_CHANGE", data.AmenitiesFromCivics, "LOC_CATEGORY_CIVICS_NAME");
     end
-    --Entertainment
+    
+    -- Entertainment
     CQUI_BuildAmenityBubbleInstance("ICON_PROJECT_CARNIVAL", data.AmenitiesFromEntertainment, "LOC_CQUI_CITY_ENTERTAINMENT");
-    --Great People
+    
+    -- Great People
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_GREAT_PEOPLE") then
         CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_CLAIM_GREAT_PERSON", data.AmenitiesFromGreatPeople, "LOC_PEDIA_CONCEPTS_PAGEGROUP_GREATPEOPLE_NAME");
     end
-    --Relgion
+
+    -- City-States
+	if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_CITY_STATES") then
+        if data.AmenitiesFromCityStates > 0 then
+            CQUI_BuildAmenityBubbleInstance("ICON_GREAT_PERSON_CLASS_WRITER", data.AmenitiesFromCityStates, "LOC_CITY_STATES_TITLE");
```

Comment:
ICON_CITY_STATE works here, if you prefer (this is what it looks like)

![image](https://user-images.githubusercontent.com/8787640/101239039-5e01bb00-3699-11eb-98ca-a09f67861c8f.png)

Reply:
I am ok with any Icon propositions. I didn't spend much time looking for them. The one you proposed is perfect.